src/shared/styles/layout/grid.css

```diff
@@ -0,0 +1,22 @@
+.container {
+    width: 100%;
+    padding: .75rem;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+.w-50 {
+    width: 50% !important;
+}
```

Comment:
I have an aversion to seeing !important :) but I guess here the class isn't doing much else

Reply:
I was following the same pattern as bootstrap where semantic CSS overrides everything. Although, I wasn't being consistent with the rest of the CSS, and to your point, this isn't really necessary at this point.

Reply:
Twitter Bootstrap is for sure NOT a good example of scalable css architecture - be careful trying to learn from twitter bootstrap - better to learn about bem or ITCSS
https://www.youtube.com/watch?v=1OKZOV-iLj4

Reply:
Good point.